backend/Application/Database/DatabaseSettings.cs

```diff
@@ -4,4 +4,8 @@ public class DatabaseSettings
 {
     public string ConnectionString { get; init; }
     public string ConnectionStringNodeCache { get; init; }
-}
\ No newline at end of file
+    /// <summary>
+    /// Configurable timeout for migrations. Defaults to 5 minutes.
+    /// </summary>
+    public TimeSpan MigrationTimeout { get; init; } = TimeSpan.FromMinutes(5);
```

Comment:
Why is this added in this PR?

Reply:
The migrations takes around 1 minute on my machine with mainnet restored locally. If it isn’t added I get timeouts on migration.